.github/workflows/prod-build.yml

```
@@ -155,19 +155,25 @@ jobs:
           repository: mdn/mdn-contributor-spotlight
           path: mdn/mdn-contributor-spotlight
 
+      - uses: actions/checkout@v4
+        if: ${{ ! vars.SKIP_BUILD }}
+        with:
+          repository: mdn/fred
+          path: mdn/fred
+
       - name: Setup Node.js environment
         if: ${{ ! vars.SKIP_BUILD || ! vars.SKIP_FUNCTION }}
         uses: actions/setup-node@v4
         with:
-          node-version-file: mdn/yari/.nvmrc
+          node-version-file: mdn/fred/.nvmrc
 
-      - name: Install all yarn packages
+      - name: Install all fred packages
```

Nit: Before it was yarn, not yari.

```suggestion
      - name: Install dependencies
```

---

It's not all dependencies, we have to install some yari ones below: the whole workflow needs a clean up at some point